scripts/globals/combat/physical_utilities.lua

```diff
@@ -381,12 +384,12 @@ xi.combat.physical.calculateMeleePDIF = function(actor, target, weaponType, wsAt
     ----------------------------------------
     -- Step 3: wRatio and pDif Caps (Melee)
     ----------------------------------------
-    local wRatio       = cRatio + (isCritical and 1.0 or 0)
-    local pDifUpperCap = 0
-    local pDifLowerCap = 0
-    local damageLimitPlus = actor:getMod(xi.mod.DAMAGE_LIMIT) / 100
-    local damageLimitPercent = (100 + actor:getMod(xi.mod.DAMAGE_LIMITP)) / 100
-    local pDifFinalCap = (xi.combat.physical.pDifWeaponCapTable[weaponType][1] + damageLimitPlus) * damageLimitPercent + (isCritical and 1.0 or 0) -- Added damage limit bonuses
+    local wRatio             = cRatio + (isCritical and 1 or 0)
+    local pDifUpperCap       = 0
+    local pDifLowerCap       = 0
+    local damageLimitPlus    = actor:getMod(xi.mod.DAMAGE_LIMIT) / 100
+    local damageLimitPercent = 1 + actor:getMod(xi.mod.DAMAGE_LIMITP) / 100
```

Comment:
This +1 here would mean all pdif caps are increased by 1.0

Reply:
No logic was changed

1 + X/100 = (100 + x) / 100

This is suposed to be a rate multiplier, meaning 1 == no change

Reply:
hmm needs more thought, hold on

Reply:
Nevermind, I thought that was the additive damage limit not the multiplicative. Long work day, sorry.